docs/topics/api/reviews.rst

```diff
@@ -89,12 +76,13 @@ If successful a :ref:`review object <review-detail-object>` is returned.
      Requires authentication.
 
 
-.. http:post:: /api/v3/addons/addon/(int:id|string:slug|string:guid)/reviews/
+.. http:post:: /api/v3/reviews/review/
 
+    :<json string addon: The add-on id the review applies to (required).
     :<json string|null body: The text of the review.
     :<json string|null title: The title of the review.
     :<json int rating: The rating the user wants to give as part of the review (required).
-    :<json int version: The add-on version id the review applies to.
+    :<json int version: The add-on version id the review applies to (required).
```

Comment:
I tried that a while back but wasn't 100% convinced - I thought it looked weird, because the type is also surrounded by parenthesis and is displayed just before the description.

Here is the result:

**version** (*int*) – The add-on version id the review applies to (required).

vs

**version** (*int*) – (required) The add-on version id the review applies to.